meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -2,12 +2,19 @@
 
 source /usr/sbin/resin-vars
 
+# The tmp supervisor.conf is used by the supervisor updater
+if [ -f /tmp/update-supervisor.conf ]; then
+	source /tmp/update-supervisor.conf
+else
+	source /etc/resin-supervisor/supervisor.conf
```

Comment:
Mixing tabs and 4-space in this file, please go to 4-space

Reply:
Fair.